common/flock.py

```diff
@@ -0,0 +1,64 @@
+# SPDX-FileCopyrightText: © 2024 Christian BUHTZ <c.buhtz@posteo.jp>
```

Comment:
Not the only possible method but one of the recommended methods how to add copyright and license info to each file in a repo.

Further reading: [REUSE Software](https://reuse.software/) and [SPDX](https://spdx.github.io/spdx-spec) ([ISO/IEC 5962:2021](https://www.iso.org/obp/ui/en/#iso:std:iso-iec:5962)) specifications